index.html

```diff
@@ -3,1359 +3,2011 @@
   <meta name="author" content="Sh3llcod3">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <meta name="theme-color" content="#3f51b5" />
+  <meta name="theme-color" content="#1b1b21" />
   <meta name="referrer" content="origin">
-  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
-  <link rel="stylesheet" href="https://unpkg.com/material-components-web@3.2.0/dist/material-components-web.min.css">
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" type="text/css">
-  <link rel="stylesheet" href="./Site_Code/styles.min.css" type="text/css">
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" type="text/css">
-  <script src="https://unpkg.com/material-components-web@3.2.0/dist/material-components-web.min.js"></script>
-  <script src="https://player.vimeo.com/api/player.js"></script>
-  <title>Cyber Discovery Community</title>
-  <link rel="icon" href="./Image_Assets/favicon.png">
-  <style>
-  </style>
+  <link rel="stylesheet" href="dist/bundle.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="static/normalize.min.css" type="text/css">
+  <script src="dist/bundle.js" defer></script>
+  <title>CD Discord Community | Home</title>
+  <link rel="icon" href="static/favicon.png">
   <script>
+    function openLinkHandler(siteLink){
+      const newWindow = window.open(siteLink, '_blank');
+      newWindow.opener = null;
+      newWindow.location = siteLink;
+      newWindow.focus();
+    }
   </script>
 </head>
 <body class="mdc-typography">
-  <template>
-    <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-12-desktop mdc-layout-grid__cell--span-4-phone mdc-layout-grid__cell--span-8-tablet">
-      <h3 class="mdc-list-group__subheader spaced-out-subheader" id="soundboard-track-category">Track Category</h3>
-    </div>
-  </template>
-  <template>
-    <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-3-desktop mdc-layout-grid__cell--span-4-phone mdc-layout-grid__cell--span-4-tablet">
-      <div class="mdc-card mdc-elevation-transition resources-tab-card spaced-card-mobile">
-        <div class="mdc-typography mdc-card__primary-action mdc-typography--subtitle1 elevated-soundboard-surface unselectable-text" tabindex="0" id="soundboard-track-name">Track Name</div>
-      </div>
+  <header class="mdc-top-app-bar app-bar mdc-top-app-bar--fixed" id="app-bar">
+    <div class="mdc-top-app-bar__row">
+      <section class="mdc-top-app-bar__section mdc-top-app-bar__section--align-start">
+        <button class="material-icons mdc-top-app-bar__navigation-icon mdc-icon-button">menu</button>
+        <span class="mdc-top-app-bar__title">CD Community</span>
```

Comment:
```suggestion
        <span class="mdc-top-app-bar__title">Cyber Discovery Community</span>
```

Reply:
I could do this, but it clips on smaller viewports